pyccel/codegen/printing/ccode.py

```diff
@@ -2272,6 +2274,9 @@ def _print_Program(self, expr):
     def _print_CustomDataType(self, expr):
         return "struct " + expr.name
 
+    def _print_Del(self, expr):
+        return ""
```

Comment:
This should call `__del__` to free memory (although maybe an issue should be opened for this as `del` probably ought to be translated to `Deallocate` in the semantic stage

Reply:
Fixed f31858d2e5402008754c6cf25de3381f8c0bca94.